config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -234,7 +234,7 @@ presubmits:
             - --
             - --build=quick
             - --cluster=
-            - --env=KUBE_FEATURE_GATES=DisableCloudProviders=false,DisableKubeletCloudCredentialProviders=false
+            - --env=KUBE_FEATURE_GATES=DisableCloudProviders=true,DisableKubeletCloudCredentialProviders=true
```

Comment:
Can you please set `CLOUD_PROVIDER_FLAG` to `external` as well? So it's consistent with the 2 disable flags?

Reply:
all the locations below as well